### DISCUSSION TITLE
Does the software also work on Quansheng UV-5R Plus?

### DISCUSSION THREAD

**Discussion by wmarkow** [egzumer/uv-k5-firmware-custom#109] [Q&A] [answered]
As in the title. Does it work?

In this [issue comment](https://github.com/egzumer/uv-k5-firmware-custom/issues/60#issuecomment-1783920238) it is written by @IT9-TOP that he has two 5R_plus devices and both of the suffer the issue #60. I silently assume that he flashed the uv-k5-firmware-custom software also to ~~UK-5R~~ UV-5R Plus and basically it works. @IT9-TOP, can you confirm?

**Comment by ErikS-web** [comment]
Pse check... I quess you mean U**V**-5R plus

**Comment by ErikS-web** [comment, marked as answer]
Read ['explanation-of-differences-between-product-types'](https://github.com/spm81/Quansheng_UV-K5#explanation-of-differences-between-product-types)

> [fcc.report.FCC-ID.XBPUV-K5.6401581.pdf](https://github.com/egzumer/uv-k5-firmware-custom/files/13326497/fcc.report.FCC-ID.XBPUV-K5.6401581.pdf)
> 
> _Subject: Explanation of differences between product types
> FCC ID : XBPUV-K5
> Model Number: **UV-K5, UV-K5(2), UV-K5(6), UV-K5(8), UV-K5(9), UV-K5(11),
> UV-K5(22), UV-K5(66), UV-K5(88), UV-K5(99), UV-5R, UV-5R PLUS, UV-82**
> To Whom It May Concern:
> We, The QUANSHENG ELECTRONICS CO., LTD. hereby declare that the models are
> identical except the model names & cover designs.
> Except for the different above, no other modification is performed._
> 
> 

This can be used on all the mentioned types and [here is the how2Flash](https://github.com/egzumer/uv-k5-firmware-custom/wiki/Flashing-the-firmware).

**Comment by wmarkow** [comment]
Thanks for your fast answer. Yes, I mean UV-5R Plus. I have one of this under my hand, but it is not mine. I borrowed for tests and need to return. I will not flash this particular device.

I did a picture of the label which is hidden under the battery. FCC ID is XBPUV-K5:
<img src="https://github.com/egzumer/uv-k5-firmware-custom/assets/14165380/6129fc0b-a3e8-4d90-aecb-e25edc42c96f" width=50% height=50%>

**Reply by ErikS-web** [reply]
You can check with Chirp what firmware is loaded

**Reply by OE1MWW** [reply]
I own a UV-K5(8) and a UV-5R Plus. On both devices the egzumer_v0.17.packed.bin is loaded and the same settings (codeplug) is used. Absolutely no difference and no problems whatsoever.

**Reply by f5gkw** [reply]
same here . 73 F5GKW.
I had reversed to v0.16

**Comment by wmarkow** [comment]
Thank you guys.
Also here https://github.com/spm81/Quansheng_UV-K5/tree/main/Firmware/UV-5R%20Plus the list of firmwares for **UV-5R Plus**.
